# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../prima/x11base.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- Prima-1.20/Makefile.PL.vanilla	2006-08-21 15:26:38.000000000 +0200
+++ Prima-1.20/Makefile.PL	2006-08-21 15:21:12.000000000 +0200
@@ -1298,6 +1298,7 @@
 		# find X11 libraries
 		my $libpath = find_lib( "X11", '', 
 			(defined($X11BASE) ? "$X11BASE/lib" : ()), 
+			(defined($X11BASE) ? "$X11BASE/lib64" : ()),
 			"/usr/X11R6/lib", 
 			"/usr/X11/lib",
 			"/usr/X/lib",
